cmake/CreateAutogeneratedCodeTargets.cmake.in

```diff
@@ -10,8 +10,13 @@ add_generated_code_lib(MODELNAME ${MDL_NAME})
 # Set the target name for the custom driver
 set(CODER_MAIN "CoderMain_${MDL_NAME}")
 
+configure_file(
+    "${CMAKE_SOURCE_DIR}/src/driver.cpp.in"
```

Comment:
In general I think it is a good idea to either use `PROJECT_SOURCE_DIR` to refer to the latest directory where the `project` was called, or `CMAKE_CURRENT_SOURCE_DIR` to refer to this directory. 
`CMAKE_SOURCE_DIR` would break if this repo is included in another CMake build using `FetchContent`/``add_subdirectory`, see https://github.com/robotology/ycm/issues/162 .

Reply:
`PROJECT_SOURCE_DIR` seems more appropriate, good catch! Thanks